addons/point_of_sale/__manifest__.py

```diff
@@ -103,12 +103,22 @@
             ('remove', 'web/static/src/core/errors/error_handlers.js'),
             ('remove', 'web/static/src/legacy/legacy_rpc_error_handler.js'),
         ],
+        
         # This bundle includes the main pos assets.
         'point_of_sale.assets': [
+            ('include', 'web._assets_helpers'),
+            'web/static/src/libs/bootstrap/pre_variables.scss',
+            'web/static/lib/bootstrap/scss/_variables.scss',
+            'web/static/src/legacy/scss/bootstrap_overridden.scss',
+            ('include', 'web._assets_bootstrap'),
+            'web/static/src/**/**/*.variables.scss',
+            'web/static/src/webclient/webclient.scss',
+
             'web/static/fonts/fonts.scss',
             'web/static/src/libs/fontawesome/css/font-awesome.css',
             'web/static/lib/daterangepicker/daterangepicker.css',
             'point_of_sale/static/src/css/pos.css',
+            'point_of_sale/static/src/scss/pos.scss',
```

Comment:
I'd go for a main `pos.variables.scss` ( and eventually `pos.functions.scss`, `pos.mixins.scss`...).
The rest of the SCSS should be split in different files organised by component.. .what do you think ?